R/characterization-incidence.R

```diff
@@ -17,6 +17,92 @@
 # limitations under the License.
 
 
+as_ggplot <- function(x){
+  # Open null device to avoid blank page before plot------
+  # see cowplot:::as_grob.ggplot
+  null_device <- base::getOption(
+    "ggpubr.null_device",
+    default = cowplot::pdf_null_device
+  )
+  cur_dev <- grDevices::dev.cur()
+  # Open null device to avoid blank page before plot
+  null_device(width = 6, height = 6)
+  null_dev <- grDevices::dev.cur()
+  on.exit({
+    grDevices::dev.off(null_dev)
+    if (cur_dev > 1) grDevices::dev.set(cur_dev)
+  })
+  # Convert to ggplot-------------
+  cowplot::ggdraw() +
```

Comment:
Just confirming that this package is added to develop?

Adding it was necessary as pulling the legend out of the plot and placing it below was very difficult without this.